WordPress/Classes/ViewRelated/Stats/Shared Views/StatsTotalRow.swift

```diff
@@ -50,9 +57,12 @@ class StatsTotalRow: UIView, NibLoadable {
     @IBOutlet weak var disclosureStackView: UIStackView!
     @IBOutlet weak var disclosureButton: UIButton!
 
+    private var rowData: StatsTotalRowData?
     private var dataBarMaxWidth: Float = 0.0
     private typealias Style = WPStyleGuide.Stats
 
+    var delegate: StatsTotalRowDelegate?
```

Comment:
Should we make this `weak`?

Reply:
Wow. It seems I failed on all the delegates. 🤦‍♂️ I've fixed this one. I'll go back and review the others and look at the `StatsTotalRows` retention. Thank you so much for noticing that!!